templates.rst

```diff
@@ -705,6 +705,11 @@ provided by Symfony:
                     site_name: 'ACME'
                     theme: 'dark'
 
+                # add headers in response
+                headers:
+                    Content-Type: 'image/svg+xml'
+                    foo: 'bar'
+
     .. code-block:: xml
 
         <!-- config/routes.xml -->
```

Comment:
Can you also update xml and php examples ?

Reply:
done :)